superset/README.md

```diff
@@ -11,17 +11,73 @@ To install Superset add the following to the `kfctl` yaml file.
 
 ```
   - kustomizeConfig:
-      parameters:
-      # Note: The admin username is admin
-      - name: SUPERSET_ADMIN_PASSWORD
-        value: admin
       repoRef:
         name: manifests
         path: superset
     name: superset
 ```
 
-By default the user and password to the Superset portal is admin/admin. You can change the password by changing the value of the `SUPERSET_ADMIN_PASSWORD`. To launch the portal, go to the routes in the namespace you installed Open Data Hub and click on the route with `superset` name.
+By default the user and password to the Superset portal is admin/admin. To launch the portal, go to the routes in the namespace you installed Open Data Hub and click on the route with `superset` name.
+
+### Parameters
+
+There are 11 parameters exposed via KfDef.
+
+#### storage_class
+
+Name of the storage class to be used for PVC created by Hue's database. This requires `storage-class` **overlay** to be enabled as well to work.
+#### superset_secret
+
+The secret containing the environment variables to set the admin credentials and the Superset app secret key used to encrypt information in the database. If not specified, environment variables from [`superset`](base/secret-superset.yaml) will be used.
+
+When creating a custom secret, the following information should be added:
+
+* **SUPERSET_ADMIN_USER**: The username of the Superset administrator. If using the `superset` secret, `admin` will be used
```

Comment:
Should this instead say "If not using the `superset` secret"?

Reply:
Actually, that's what I meant: If you use the default secret you will get these values set by default.

Do you think I should reword this?

Reply:
I think I'd rephrase it to say "If using the default `superset` secret,"....